src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px $card-padding $card-padding;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: $main-black;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+    text-align: center;
+  }
+
+  &__title {
+    @extend %large-text;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-family: $main-font;
+    font-weight: 400;
+    line-height: 14px;
+    font-size: 10px;
+    color: $main-gray;
+    margin: 0 0 16px;
+  }
+
+  &__rating {
+    @extend %flex-sb-center;
+    margin: 0 0 24px;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__price {
+    @extend %flex-sb-center;
+    margin: 0 0 16px;
+  }
+
+  &__property {
+    margin: 0;
+    font-size: 12px;
+    color: $main-gray;
+  }
+
+  &__amount {
+    line-height: 18px;
+    font-weight: 700;
+    margin: 0;
+    font-size: 16px;
+  }
+
+  &__button {
+    display: block;
+    box-sizing: border-box;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    padding-top: 10px;
+    padding-bottom: 11px;
+    background-color: $main-blue;
+    border-radius: 5px;
+    border: 1px solid transparent;
+  }
+
+  &__button:hover {
+    color: $main-blue;
+    background-color: #fff;
+    border: 1px solid $main-blue;
```

Comment:
move border color yo your button styles and you don't need to change it on hover